sdk/ml/azure-ai-ml/azure/ai/ml/_arm_deployments/arm_templates/workspace_param.json

```diff
@@ -143,9 +143,6 @@
     "offline_store_subscription_id" : {
         "value": ""
     },
-    "materialization_identity_name": {
```

Comment:
why others have value "" and this one just removed? is there difference?

Reply:
because the arm template's validation can't tolerant empty string for this identity provision.